R/readWriteMgfData.R

```diff
@@ -45,13 +60,17 @@ writeMgfDataFile <- function(splist, con, COM = NULL, TITLE = NULL,
     if (verbose)
       setTxtProgressBar(pb, i)
 
-    writeMgfContent(splist[[i]], TITLE = NULL, con = con)
+    if (haveAddFields)
+        writeMgfContent(splist[[i]], TITLE = NULL, con = con,
+                        addFields = addFields[i, ])
```

Comment:
I know it is not your fault because it is already part of the original code but `TITLE = NULL` overwrites the `TITLE` argument. Seems like a bug nobody found/mentioned yet.

Reply:
good point! I'll fix.

Reply:
Strange. In fact I **did** fix it. Don't know why it does not show above.